link/src/main/java/com/stripe/android/link/model/Navigator.kt

```diff
@@ -41,10 +41,12 @@ internal class Navigator @Inject constructor() {
     /**
      * Behaves like a back button, popping the back stack and dismissing the Activity if this was
      * the last screen.
-     * Only performs any action if [backNavigationEnabled] is true.
+     * When [userInitiated] is true, only performs any action if [userNavigationEnabled] is true.
+     *
+     * @param userInitiated Whether the action was initiated by user interaction.
      */
-    fun onBack() {
-        if (backNavigationEnabled) {
+    fun onBack(userInitiated: Boolean = false) {
```

Comment:
I’m not sure if this should have a default value. It could result in someone calling this method without realizing that it takes a default argument, which could lead to unintended behavior.